bloom-instance/network/subnets/main.tf

```diff
@@ -0,0 +1,8 @@
+
+//data "aws_default_tags" "current" {}
+
+locals {
+  add_ngw = var.gateway_type == "ngw" ? true : false
```

Comment:
nit/offhand thought, non-blocking: idk if this works in .tf but `add_ngw = (var.gateway_type == "ngw")` is how i generally do this

Reply:
Terraform has some pretty janky conditionals.  Unfortunately, it's pretty much just ternary or nothing in this case.